lib/workers/repository/update/pr/pr-reuse.ts

```diff
@@ -4,11 +4,10 @@ import { logger } from '../../../../logger';
 import { platform } from '../../../../modules/platform';
 import type { Pr } from '../../../../modules/platform/types';
 
-const REOPEN_THRESHOLD_MILLIS = 1000 * 60 * 60 * 24 * 7;
+const REOPEN_THRESHOLD_MILLIS = 1000 * 60 * 60 * 24 * 7; // 7 days
```

Comment:
Would a change like:


```suggestion
const ONE_DAYS_MILLIS = 1000 * 60 * 60 * 24;
const REOPEN_THRESHOLD_MILLIS = ONE_DAYS_MILLIS * 7;
```

be reasonable in the style we follow in the Renovate codebase, as well as what could be seen as more idiomatic Typescript?

(genuinely unsure - looking for your experience and input!)

Reply:
Hey, I randomly ended up here, but I've thought about this exact same problem and this solution feels readable and nice to use, particularly the lsp hints.

> ```ts
> enum Duration {
>     Millisecond = 1,
>     Second = Duration.Millisecond * 1000,
>     Minute = Duration.Second * 60,
>     Hour = Duration.Minute * 60,
>     Day = Duration.Hour * 24,
>     Week = Duration.Day * 7
> }
> 
> const REOPEN_THRESHOLD_MILLIS = Duration.Week;
> // ^ 604800000
> ```
> — [ts-playground-link](https://www.typescriptlang.org/play/?#code/KYOwrgtgBAImBOBDALgSwPYigbwFBQKgFlUAbU1AZ2AGNMATKAXigEYAafQgZVoedgIUGEADoS5KnxCMAVGwAMSzoWKpwyYALhI0mUbzoyo8gGwKVhABLoE2oXrEkNWsxa4EYiAJ73dI0Rs7eQAmABZLAgB1YGAAaz9hfS9feQB2XABfXFwjSmQoACUAUQB5AAVigDkAfQAVKxLuK1KAGRgaogBJVtau7kTHURj4gG5cAHoJqAA9KHMwgA4lFaA)
> 


<img width="371" height="54" alt="Screenshot 2025-10-05 at 14 45 59" src="https://github.com/user-attachments/assets/8a7ec8b5-9ae0-40b0-8f47-f62297416cf2" />
<img width="384" height="83" alt="Screenshot 2025-10-05 at 14 45 49" src="https://github.com/user-attachments/assets/27aa3fe2-f78b-4c43-a26b-a7a6345295c8" />

Reply:
I like the above idea by @erikjuhani. But since this change is unrelated to this PR I will create a separate one for it